po/pl.po

```diff
@@ -0,0 +1,211 @@
+msgid ""
+msgstr ""
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: POEditor.com\n"
+"Project-Id-Version: WhatsApp for Linux\n"
+"Language: pl\n"
+
+#: ../src/ui/MainWindow.cpp:343 ../src/ui/TrayIcon.cpp:46
+#: ../resource/ui/MainWindow.ui.h:4
+msgid "O programie"
+msgstr "Über"
+
+#: ../src/ui/MainWindow.cpp:345 ../resource/ui/MainWindow.ui.h:6
+msgid "WhatsApp for Linux"
+msgstr "WhatsApp dla Linux"
+
+#: ../src/ui/MainWindow.cpp:346
+msgid "An unofficial WhatsApp desktop application for Linux"
+msgstr "Nieoficjalna aplikacja WhatsApp dla Linuksa"
+
+#: ../src/ui/PreferencesWindow.cpp:40
+msgid "On Demand"
+msgstr "Na życzenie"
+
+#: ../src/ui/PreferencesWindow.cpp:41
+msgid "Always"
+msgstr "Zawsze"
+
+#: ../src/ui/PreferencesWindow.cpp:42
+msgid "Never"
+msgstr "Nigdy"
+
+#: ../src/ui/TrayIcon.cpp:45
+msgid "Show"
+msgstr "Pokaż"
+
+#: ../src/ui/TrayIcon.cpp:47 ../resource/ui/MainWindow.ui.h:5
+#: ../resource/ui/ShortcutsWindow.ui.h:10
+msgid "Quit"
+msgstr "Wyjdź"
+
+#: ../src/ui/WebView.cpp:75
+msgid "Save File"
+msgstr "Zapisz Plik"
+
+#: ../src/ui/WebView.cpp:76
+msgid "Ok"
+msgstr "Ok"
+
+#: ../src/ui/WebView.cpp:77
+msgid "Cancel"
+msgstr "Anuluj"
+
+#: ../src/ui/WebView.cpp:328
+msgid "Unresponsive"
+msgstr "Nie odpowiada
+#: ../src/ui/WebView.cpp:329
```

Comment:
Could you add a newline after this? Probably, the build fails because of this

Reply:
Done